### PR TITLE
Add Time Series panel from 10.0.3

### PR DESCRIPTION
**Pull request by mikhail-vl**
Add Time Series panel from 10.0.2/10.0.3.

### CODE REVIEW

**Comment by asimonok** [comment]
Works with `id: volkovlabs-timeseries-panel` in plugin.json

Screenshot 2023-09-01 at 11 32 48

File: src/app/features/explore/utils/links.ts [hunk 1]
```diff
@@ -19,15 +19,16 @@ import {
...
 import { getTransformationVars } from 'app/features/correlations/transformations';
 
 import { getLinkSrv } from '../../panel/panellinks/link_srv';
 
 type DataLinkFilter = (link: DataLink, scopedVars: ScopedVars) => boolean;
 
 const dataLinkHasRequiredPermissionsFilter = (link: DataLink) => {
-  return !link.internal || contextSrv.hasAccessToExplore();
+  return !link.internal;
+  // return !link.internal || contextSrv.hasAccessToExplore();
```
Comment (asimonok) [inline src/app/features/explore/utils/links.ts:31]: Commented

File: src/app/features/panel/panellinks/link_srv.ts [hunk 2]
```diff
@@ -0,0 +1,355 @@
...
+  getLinkUrl: (link: any) => string;
+}
+
+export class LinkSrv implements LinkService {
+  getLinkUrl(link: any) {
+    let url = locationUtil.assureBaseUrl(getTemplateSrv().replace(link.url || ''));
+    let params: { [key: string]: any } = {};
+
+    if (link.keepTime) {
+      // const range = getTimeSrv().timeRangeForUrl();
```
Comment (asimonok) [inline src/app/features/panel/panellinks/link_srv.ts:261]: commented

File: src/app/features/variables/getAllVariableValuesForUrl.ts [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+    const scopedVar = scopedVars && scopedVars[variable.name];
+
+    if (variable.skipUrlSync) {
+      continue;
+    }
+
+    if (scopedVar) {
+      params[VARIABLE_PREFIX + variable.name] = scopedVar.value;
+    } else {
+      // params[VARIABLE_PREFIX + variable.name] = variableAdapters.get(variable.type).getValueForUrl(variable as any);
```
Comment (asimonok) [inline src/app/features/variables/getAllVariableValuesForUrl.ts:22]: commented

File: src/app/features/panel/panellinks/link_srv.ts (hunk 2 above)
Comment (mikhail-vl) [reply src/app/features/panel/panellinks/link_srv.ts:261]: not used

File: src/app/features/variables/getAllVariableValuesForUrl.ts (hunk 3 above)
Comment (mikhail-vl) [reply src/app/features/variables/getAllVariableValuesForUrl.ts:22]: not used

**Review by mikhail-vl — COMMENTED** [review]
Looks great!

**Pull request merged**